### DISCUSSION TITLE
Possible to join all Meshes?

### DISCUSSION THREAD

**Discussion by arpu** [donmccurdy/glTF-Transform#212] [Q&A] [answered]
Hello,

is it possible to join ( like in Blender)  all meshes ( threejs mergeBufferGeometries)

**Comment by donmccurdy** [comment, marked as answer]
It's possible to write a script that does all of this, applying [`node.getWorldMatrix()`](https://gltf-transform.donmccurdy.com/classes/node.html#getworldmatrix) to each [Mesh](https://gltf-transform.donmccurdy.com/classes/mesh.html) and then merging the vertex attribute arrays, but there is no out-of-the-box function provided to automatically do that now. The tracking issue for this idea is https://github.com/donmccurdy/glTF-Transform/issues/11.
